pom.xml

```diff
@@ -37,12 +37,12 @@
         <junit.version>4.13.1</junit.version>
         <assertj.version>3.24.2</assertj.version>
 
-        <jsp.api.version>2.3.6</jsp.api.version>
-        <servlet.api.version>4.0.4</servlet.api.version>
-        <org.springframework.version>5.3.36</org.springframework.version>
-        <spring-security.version>5.8.12</spring-security.version>
+        <jsp.api.version>4.0.0-M2</jsp.api.version>
+        <servlet.api.version>6.1.0-M2</servlet.api.version>
```

Comment:
what's M2 in the version?

Reply:
> what's M2 in the version?

M2 was the latest version at the time PR is created. Now new versions have been released.

rev : efd7ad25f83f0a47fca346edb8b8bef22c908acd